src/modules/answer/answer.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Answer } from '../database/entities/answer.entity';
+import { AnswerDto } from '../database/dto/answer.dto';
+
+@Injectable()
+export class AnswerService {
+  constructor(
+    @InjectRepository(Answer)
+    private readonly answerRepository: Repository<Answer>,
+  ) {}
+
+  async getAnswers(): Promise<AnswerDto[]> {
+    return this.answerRepository.find();
+  }
+
+  async getAnswerById(id: string): Promise<AnswerDto> {
+    return this.answerRepository.findOne(id);
+  }
+
+  async createAnswer(answer: AnswerDto): Promise<AnswerDto> {
+    const newAnswer = await this.answerRepository.create(answer); // crypto in Node
+    await this.answerRepository.save(newAnswer);
+    return newAnswer;
+  }
+
+  async updateAnswer(id: string, answer: AnswerDto): Promise<AnswerDto> {
+    await this.answerRepository.update(id, answer);
+    return this.answerRepository.findOne(id);
+  }
```

Comment:
update with returning

Reply:
done
I used: return this.userRepository.save({ id, user });